src/Temporalio/Worker/TemporalWorker.cs

```diff
@@ -102,9 +104,39 @@ public TemporalWorker(IWorkerClient client, TemporalWorkerOptions options)
         public TemporalWorkerOptions Options { get; private init; }
 
         /// <summary>
-        /// Gets the client this worker was created with.
+        /// Gets or sets the client for this worker.
         /// </summary>
-        internal IWorkerClient Client { get; private init; }
+        /// <remarks>
+        /// When this property is set, it actually replaces the underlying client that is being used
+        /// by the worker. This means the next calls by the worker to Temporal (e.g. responding
```

Comment:
nitpicky

```suggestion
        /// by the worker. This means subsequent calls by the worker to Temporal (e.g. responding
```

Reply:
:+1: This one in particular may not be worth another commit.